client/src/components/MapElement/MapElement.js

```diff
@@ -31,7 +31,9 @@ export function MapElement({ className }) {
   }, [appView])
 
   return (
-    <div className={`${bgUrl} bg-cover bg-center ${className}`}>
+    <div
+      className={`${bgUrl} cursor-move bg-cover bg-center hover:brightness-95 ${className}`}
```

Comment:
why is this cursor move btw?

Reply:
Good catch, it really doesn't have to be except just as a visual indicator of "where you can drag the map" (after Andy adds mapbox, which has its own unique cursor) vs "where dragging will have no effect. 

But with the current design you can tell even if you don't have the cursor move thing so I will remove this.

Reply:
Done